CodeStarterCamp_Week3/main.swift

```diff
@@ -18,18 +18,22 @@ enum Coffee: String {
 }
 
 class Person {
+    var name: String
     var money: Int
     var job: String?
     var house: String?
     
-    init(money: Int) {
+    init(name: String, money: Int) {
+        self.name = name
         self.money = money
     }
     
-    func buyCoffee(paying: Int) {
-        print("지불할 돈은 \(paying)입니다")
-        self.money -= paying
-        print("남은 돈은 \(self.money)입니다")
+    func buyCoffee(paying: Int) -> Int {
+        let subtractMoney = self.money - paying
+        if subtractMoney >= 0 {
```

Comment:
훨씬 간단해졌네요 👍

```suggestion
        
        if subtractMoney >= 0 {
```

상수 선언과 `if` 사이에 개행이 있으면 가독성이 더 좋을 것 같네요!